src/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```

Comment:
2018 and elsewhere

Reply:
We're not supposed to change the copyright date

Reply:
It's a new file.